### ISSUE TITLE
Java players become invisible to Bedrock players

### ISSUE THREAD

**Issue by Comstepr** [GeyserMC/Geyser#1295]
**Describe the bug**
Java players become invisible to Bedrock players

**To Reproduce**
1. Have a Java player teleport to a bedrock player from the nether
2. Have a Bedrock player teleport to a java player from the nether

**Expected behavior**
To work as intended and still be visible

**Screenshots / Videos**
Unable to obtain as reported from my playerbase

**Server Version**
This server is running Tuinity version git-Tuinity-"ef4e59a" (MC: 1.16.3) (Implementing API version 1.16.3-R0.1-SNAPSHOT)
Checking version, please wait...
Previous version: git-Tuinity-"fe40e04" (MC: 1.16.3)
You are running the latest version

**Geyser Version**
This server is running Geyser version 1.1.0 (git-master-3dec200) (Java: 1.16.3, Bedrock: 1.16.0 - 1.16.100)
Checking version, please wait...
Geyser-Spigot
**This is my fork of Geyser. I will not be verifying this bug on the official Geyser file as I do not use the official Geyser jar file.**

**Minecraft: Bedrock Edition Version**
1.16.3

**Additional Context**
https://dump.geysermc.org/PB3CPnJZsKdrzhzGtOY9V0CrSAHPATuo

**Comment by zimzaza4** [comment]
I also encountered this problem and did not use the official geyser

**Comment by RednedEpic** [comment]
Can you (or someone else) please provide a video of this happening on the _official_ Geyser? We do not support forks. Additionally, just as a rule of thumb, if you cannot replicate this yourself, please do not open an issue here unless you are able to replicate it on your own. Will be closing this issue until the above are done.

**Comment by MrZomka** [comment]
Had the same issue. Updating resolved the issue.

**Comment by Comstepr** [comment]
> Can you (or someone else) please provide a video of this happening on the _official_ Geyser? We do not support forks. Additionally, just as a rule of thumb, if you cannot replicate this yourself, please do not open an issue here unless you are able to replicate it on your own. Will be closing this issue until the above are done.

As I mentioned **I will not be verifying this bug on the official Geyser file as I do not use it myself**. We report bugs **to be helpful and not all the time it is for our own benefit**. If this issue does not want to be fixed by the developer, then so be it.

**It is a very irrelevant thing to say "use the official Geyser" when clearly, my fork is literally on par with the branch.**



> Had the same issue. Updating resolved the issue.
My fork is **fully updated**.

>  if you cannot replicate this yourself, please do not open an issue here unless you are able to replicate it on your own
So if I can't, this issue will remain unfixed.

The point is, not all the time issues are **opened for our own benefit.** If no one wants to solve the problem, then I guess it no longer has **anything to do with us and it is more of how the team at Geyser wants to handle this issue, with all due respect. You have a bunch of contributors and developers as well as a test server, surely you could get them to test this issue out for you.**

**Comment by MrZomka** [comment]
> **as I do not use it myself**
> It is a very irrelevant thing to say "use the official Geyser" when clearly, my fork is literally on par with the branch.

Why did you do a bug report? It's your fork - your problem smh

**Comment by Comstepr** [comment]
> > **as I do not use it myself**
> > It is a very irrelevant thing to say "use the official Geyser" when clearly, my fork is literally on par with the branch.
> 
> Why did you do a bug report? It's your fork - your problem smh

I expect the bug report to **benefit Geyser themselves.** If you have nothing nice to say then please do not say it in a public say, with all due respect.

**Comment by MrZomka** [comment]
I think reporting a bug in a fork that you did isn't nice. You didn't even check if it's your fault.

**Comment by Comstepr** [comment]
Ok - here's what happened:
I've spoken to Camotoy on Discord about this situation. I will only be replying if necessary. Otherwise if this issue persists I will let it persist.

Thank you everyone.

**Comment by toinouH** [comment]
It is not because a fork is updated that the modifications made by the owner of repo can't create bugs. 
If you want a fix the issue has be to _official_ (it means reproduced with the official version)
Thank you for reading :)

**Comment by Comstepr** [comment]
> It is not because a fork is updated that the modifications made by the owner of repo can't create bugs.
> If you want a fix the issue has be to _official_ (it means reproduced with the official version)
> Thank you for reading :)

I'm not arguing any further on this. My fork just uploads the Geyser file to the repo itself for me to auto-download it. As I mentioned I have spoke to Camotoy. If anyone is facing this issue themselves I suggest you make a new issue.

**Comment by MrZomka** [comment]
Basically, you compiled Geyser yourself?
_why not just use the official jenkins_

**Comment by RednedEpic** [comment]
In the future for those who participated, please leave irrelevant discussion out of issues. 

When opening bug reports in the future, please make sure you are using the _official_ Geyser version. Even if you only made one tweak here and there, we have no guarantee that the tweak you may have made is causing the bug or not, and other factors such as your fork being outdated may come into play, which is why we require you report using the official Geyser jar. Although you stated this may not be the case for your build, again, we have no guarantees on our end. It makes it much easier for us if you first verify if the problem happens on an official Geyser jar.

Additionally, if you cannot replicate the issue yourself, don't open one. One of the purposes of this tracker is to track bugs which can be replicated. This is not the place to open issues of things that "might be" bugs, and asking us to test that ourselves defeats the entire purpose of this issue tracker. If you don't know whether something is a bug or not, please do feel free to reach out to us on Discord and we'll be happy to help you there.
